testing/path.go

```diff
@@ -39,47 +41,57 @@ func (path *Path) SetChannelOrdered() {
 // if EndpointA does not contain a packet commitment for that packet. An error is returned
 // if a relay step fails or the packet commitment does not exist on either endpoint.
 func (path *Path) RelayPacket(packet channeltypes.Packet) error {
+	_, _, err := path.RelayPacketWithResults(packet)
+	return err
+}
+
+// RelayPacketWithResults attempts to relay the packet first on EndpointA and then on EndpointB
+// if EndpointA does not contain a packet commitment for that packet. The function returns:
+// - The result of the packet receive transaction.
+// - The acknowledgement written on the receiving chain.
+// - An error if a relay step fails or the packet commitment does not exist on either endpoint.
+func (path *Path) RelayPacketWithResults(packet channeltypes.Packet) (res *sdk.Result, ack []byte, err error) {
```

Comment:
I don't think we should use named returns, I think these are pretty error prone. The name of the variables in the method body should be clear enough to indicate their purpose.